src/privatesend/privatesend-client.cpp

```diff
@@ -728,15 +728,16 @@ bool CPrivateSendClientManager::CheckAutomaticBackup()
     if (!pwalletMain) {
```

Comment:
Maybe out of scope of this PR, but I have the feeling that this check and the corresponding `= false` should happen at inititialization time instead of here and this location should just check for `!fEnablePrivateSend || !fPrivateSendRunning`

Reply:
cbf7caef629306246d979ecbb54da8e503b5efac kind of does the init part but I pushed more replacement for `!pwalletMain` just now.